### PR TITLE
FAI-1118 Page to preview the application answers

### PR DESCRIPTION
**Pull request by balaji-j**
Page to preview the application answers before submit

Query and Command handlers to Get the Application Summary and Update the Application Status

Story: https://skillsfundingagency.atlassian.net/browse/FAI-1118

### CODE REVIEW

File: src/SFA.DAS.FAA.Web/Models/Apply/ApplicationSummaryViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    }
+
+    public bool IsDisabilityConfident { get; init; }
+
+    public bool IsApplicationComplete => EducationHistory is {TrainingCoursesStatus: SectionStatus.Completed, QualificationsStatus: SectionStatus.Completed} &&
+                                         WorkHistory is {VolunteeringAndWorkExperienceStatus: SectionStatus.Completed, JobsStatus: SectionStatus.Completed} &&
+                                         (ApplicationQuestions.AdditionalQuestion1Id is not null || ApplicationQuestions.AdditionalQuestion1Status == SectionStatus.Completed) &&
+                                         (ApplicationQuestions.AdditionalQuestion2Id is not null || ApplicationQuestions.AdditionalQuestion2Status == SectionStatus.Completed) &&
+                                         InterviewAdjustments.RequestAdjustmentsStatus == SectionStatus.Completed &&
+                                         (!IsDisabilityConfident || DisabilityConfidence.InterviewUnderDisabilityConfidentStatus == SectionStatus.Completed);
```
Comment (dashton82) [inline src/SFA.DAS.FAA.Web/Models/Apply/ApplicationSummaryViewModel.cs:90]: One for later but I feel this should come from the candidate API, it knows all the statuses and can also determine if the application can be submitted.

File: src/SFA.DAS.FAA.Web/Models/Apply/ApplicationSummaryViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        public Guid Id { get; set; }
+        public string? GovUkIdentifier { get; set; }
+        public string? Email { get; init; }
+        public string FullName => $"{FirstName} {LastName}";
+        private string? FirstName { get; init; }
+        public string? MiddleName { get; set; }
+        private string? LastName { get; init; }
+        public string? PhoneNumber { get; init; }
+        private DateTime? DateOfBirth { get; init; }
+        public string BirthDate => DateOfBirth.HasValue ? DateOfBirth.Value.ToString("dd MMM yyyy") : string.Empty;
```
Comment (dashton82) [inline src/SFA.DAS.FAA.Web/Models/Apply/ApplicationSummaryViewModel.cs:40]: Can we just double check that we should be showing this - I know we capture but I don't think its part of the application.

**Review by dashton82 — APPROVED** [review]
All good - just a couple of comments to remind me to create some tickets and check things 😄

**Comment by balaji-j** [comment]
Already merged and changes are there in main

**Pull request closed without merging**